Feisty.xcodeproj/xcshareddata/xcschemes/Feisty.xcscheme

```diff
@@ -72,9 +72,17 @@
       </BuildableProductRunnable>
```

Comment:
The scheme was edited to to enable the use of mocks without having to comment out code. Also includes firebase debugging arguments